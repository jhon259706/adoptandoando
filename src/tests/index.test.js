import Home from '@pages/index';
import { render } from '@tests/test-utils';

describe('Home', () => {
  it('renders home', () => {
    const { container, asFragment } = render(<Home />);
    expect(asFragment()).toMatchSnapshot();
    expect(container.querySelector('#title')).toHaveTextContent('Chempopet');
  });
});
