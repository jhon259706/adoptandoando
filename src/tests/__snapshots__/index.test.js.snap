// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home renders home 1`] = `
<DocumentFragment>
  <div>
    <h1
      id="title"
    >
      Chempopet
    </h1>
    <div
      class="sc-bdnxRM hUNmRu"
    >
      <div
        class="sc-jSFjdj iIMsiI"
      >
        <div
          style="display: inline-block; max-width: 100%; overflow: hidden; position: relative; box-sizing: border-box; margin: 0px;"
        >
          <div
            style="box-sizing: border-box; display: block; max-width: 100%;"
          >
            <img
              alt=""
              aria-hidden="true"
              role="presentation"
              src="data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iNzgwIiBoZWlnaHQ9IjUyMCIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIiB2ZXJzaW9uPSIxLjEiLz4="
              style="max-width: 100%; display: block; margin: 0px; padding: 0px;"
            />
          </div>
          <noscript />
          <img
            alt="Banner image"
            decoding="async"
            src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
            style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%;"
          />
        </div>
      </div>
      <div
        class="sc-gtsrHT cuyRcv"
      >
        <div
          class="sc-dlnjwi bPagKz"
        >
          <header
            class="sc-hKFxyN bmDJH"
          >
            ÚNETE A CHEMPOPET
          </header>
          <p
            class="sc-eCApnc ghUIxW"
          >
            Creemos en un mundo mejor
          </p>
        </div>
      </div>
    </div>
    <div
      class="sc-fujyAs eTDfoZ"
    >
      <div
        class="sc-pNWdM fzXTrW"
      >
        <div
          style="display: block; overflow: hidden; position: relative; box-sizing: border-box; margin: 0px;"
        >
          <div
            style="display: block; box-sizing: border-box; padding-top: 66.66666666666666%;"
          />
          <noscript />
          <img
            alt="About Us image"
            decoding="async"
            src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
            style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%;"
          />
        </div>
        <header
          class="sc-iqAclL buoUpv"
        >
          SOBRE CHEMPOPET
        </header>
      </div>
      <div
        class="sc-jrsJWt hCkdGC"
      >
        <div
          class="sc-bqGGPW jQJFLR"
        >
          <header
            class="sc-kEqXSa lklqJw"
          >
            SOBRE CHEMPOPET
          </header>
          <p
            class="sc-crzoAE HguOt"
          >
            Establecido(a) en el año 2000
          </p>
          <p
            class="sc-dIsUp iHqDCi"
          >
            Como un(a) Organización de rescate de mascotas líder, nuestros proyectos y campañas son reconocidos en todo el mundo. Trabajamos para unir a personas de todas las edades y orígenes que comparten la creencia de un mundo en el que los animales están libres de tratos crueles e inhumanos. ChempoPet está dirigido por miembros para sus miembros, ofreciendo una voz unida para todos aquellos que quieran formar parte.
          </p>
        </div>
      </div>
      <button
        class="sc-ksluID didBIR"
        href="/"
      >
        Contacto
      </button>
    </div>
    <section
      class="sc-gKAaRy cqlArC"
    >
      <header
        class="ourservices__header"
      >
        NUESTROS SERVICIOS
      </header>
      <h3>
        ¿Qué hacemos?
      </h3>
      <div
        class="ourservices__cards"
      >
        <div
          class="sc-iCoGMd ifvnlW"
        >
          <div
            style="display: inline-block; max-width: 100%; overflow: hidden; position: relative; box-sizing: border-box; margin: 0px;"
          >
            <div
              style="box-sizing: border-box; display: block; max-width: 100%;"
            >
              <img
                alt=""
                aria-hidden="true"
                role="presentation"
                src="data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iNTAwIiBoZWlnaHQ9IjQ2MCIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIiB2ZXJzaW9uPSIxLjEiLz4="
                style="max-width: 100%; display: block; margin: 0px; padding: 0px;"
              />
            </div>
            <noscript />
            <img
              alt="Service Card Image"
              decoding="async"
              src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
              style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%;"
            />
          </div>
          <div
            class="card__description"
          >
            <h4>
              ADOPCIÓN
            </h4>
            <br />
            <p>
              Brindale un hogar a un animal
            </p>
          </div>
        </div>
        <div
          class="sc-iCoGMd cWySGA"
        >
          <div
            style="display: inline-block; max-width: 100%; overflow: hidden; position: relative; box-sizing: border-box; margin: 0px;"
          >
            <div
              style="box-sizing: border-box; display: block; max-width: 100%;"
            >
              <img
                alt=""
                aria-hidden="true"
                role="presentation"
                src="data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iNTAwIiBoZWlnaHQ9IjQ2MCIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIiB2ZXJzaW9uPSIxLjEiLz4="
                style="max-width: 100%; display: block; margin: 0px; padding: 0px;"
              />
            </div>
            <noscript />
            <img
              alt="Service Card Image"
              decoding="async"
              src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
              style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%;"
            />
          </div>
          <div
            class="card__description"
          >
            <h4>
              RESCATE Y BUSQUEDA
            </h4>
            <br />
            <p>
              Ayuda a encontrar una mascota perdida
            </p>
          </div>
        </div>
        <div
          class="sc-iCoGMd bOAfwq"
        >
          <div
            style="display: inline-block; max-width: 100%; overflow: hidden; position: relative; box-sizing: border-box; margin: 0px;"
          >
            <div
              style="box-sizing: border-box; display: block; max-width: 100%;"
            >
              <img
                alt=""
                aria-hidden="true"
                role="presentation"
                src="data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iNTAwIiBoZWlnaHQ9IjQ2MCIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIiB2ZXJzaW9uPSIxLjEiLz4="
                style="max-width: 100%; display: block; margin: 0px; padding: 0px;"
              />
            </div>
            <noscript />
            <img
              alt="Service Card Image"
              decoding="async"
              src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
              style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%;"
            />
          </div>
          <div
            class="card__description"
          >
            <h4>
              RELACIONADOS
            </h4>
            <br />
            <p>
              Encuentra servicios y productos utiles para tu mascota
            </p>
          </div>
        </div>
      </div>
    </section>
  </div>
</DocumentFragment>
`;
